frontend/src/components/progress/progress_stage_waiting.scss

```diff
@@ -51,7 +51,8 @@ h2 {
   @include common.flex-column-align-center;
   flex-wrap: wrap;
   gap: common.$spacing-medium;
-  max-width: 80px;
+  max-width: 100px;
   overflow-wrap: anywhere;
   text-align: center;
-}
\ No newline at end of file
+  font-size: var(--md-sys-typescale-body-small-font-size);
```

Comment:
Instead of setting font size, include a font bundle (which will also adjust line height, spacing, etc.), e.g., `@include typescale.body-small);`